elasticdl/python/elasticdl/image_builder.py

```diff
@@ -81,6 +79,55 @@ def build_and_push_docker_image(
     return image_name
 
 
+def remove_images(
```

Comment:
Is there an option `-f` for force removal?

Reply:
`force` option available in docker python API for cases where you want to remove an image in a force way, without considering dependencies and containers. I put `force`=False here by default.